### PR TITLE
SWS-163 Add unit test for graph generation

### PR DESCRIPTION
**Pull request by jshaughn**
This is pending the PR for a hook in client.go...

### CODE REVIEW

**Comment by jshaughn** [comment]
@jmazzitelli Review this please.

File: handlers/graph_test.go [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/gorilla/mux"
+
+	"github.com/stretchr/testify/assert"
+
```
Comment (jmazzitelli) [inline handlers/graph_test.go]: imports should have third party deps in the middle and our sws imports at the bottom with blank line between section. totally trivial and only triggering my ocd

Comment (jshaughn) [reply handlers/graph_test.go]: i will fix this on a following PR (the tests need a change based on another pending change)

File: handlers/graph_test.go [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+		mock.AnythingOfType("*context.emptyCtx"),
+		query,
+		mock.AnythingOfType("time.Time")).
+		Return(*ret, nil)
+	api.On(
+		"Query",
+		mock.AnythingOfType("*context.cancelCtx"),
+		query,
+		mock.AnythingOfType("time.Time")).
+		Return(*ret, nil)
```
Comment (jmazzitelli) [inline handlers/graph_test.go]: confusing the way this is formatted - thought Return was just another argument till I realized the line above ends with . and not ,

Comment (jshaughn) [reply handlers/graph_test.go]: i'll pull that up in the next PR

**Pull request merged**